UI/src/components/widgets/product/view.js

```diff
@@ -241,8 +205,8 @@
                         score = -1;
                     } else {
                         var metricValue = Math.round(configuredTeam.summary[metricName].number);
-                        scoreMetaData.scoreRanges.forEach(function(rangeObj) {
-                            if (metricValue >= rangeObj.rangeMin && metricValue <= rangeObj.rangeMax) {
+                        scoreMetaData.gamificationRangeScores.forEach(function(rangeObj) {
```

Comment:
I guess this should be gamificationScoringRanges. Please rebuild your API and re-test it once to make sure everything is alright.